SimPEG/NSEM/ProblemNSEM.py

```diff
@@ -0,0 +1,560 @@
+from SimPEG.EM.Utils.EMUtils import omega, mu_0
```

Comment:
I apologize for this import statement! This seems silly. What do you think of
```
from SimPEG.EM import mu_0, omega
```
?